packages/manager-react-components/src/components/datagrid/text-cell.component.tsx

```diff
@@ -3,14 +3,20 @@ import { OdsText } from '@ovhcloud/ods-components/react';
 import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
 import { useTranslation } from 'react-i18next';
 
+type DataGridTextCellProps = {
+  className?: string;
+  'data-testid'?: string;
+};
+
 /** Simple datagrid cell text formatter applying ODS style */
 export function DataGridTextCell({
   className,
   children,
-}: React.PropsWithChildren<{ className?: string }>) {
+  'data-testid': dataTestId,
+}: React.PropsWithChildren<DataGridTextCellProps>) {
   const { t } = useTranslation('datagrid');
   return (
-    <OdsText preset={ODS_TEXT_PRESET.span}>
+    <OdsText preset={ODS_TEXT_PRESET.span} data-testid={dataTestId}>
```

Comment:
question: Can it wait for MUKv1, because with ODS v19 you can directly use utils getByText and so on.And this becomes redundant.

Reply:
This for e2e tests with playwright, not unit test.
Our QA engineer wants to be able to get data on different rows of a datagrid by pointing test-ids like `[ressource-id]-urn` for example